GiniVision/Classes/Core/Multipage Review/MultipageReviewMainCollectionCell.swift

```diff
@@ -62,6 +75,52 @@ final class MultipageReviewMainCollectionCell: UICollectionViewCell {
         }
     }
     
+    func fill(with documentRequest: DocumentRequest, errorAction: @escaping (NoticeActionType) -> Void) {
```

Comment:
The name is misleading. It sounds like a purely visual property, but in reality it populates the cell.